packages/e2e-tests/puppeteer/constants.ts

```diff
@@ -5,7 +5,10 @@ export const USE_DAPPETEER = getEnvBool('USE_DAPPETEER');
 export const USES_VIRTUAL_FUNDING = process.env.REACT_APP_FUNDING_STRATEGY === 'Virtual';
 export const JEST_TIMEOUT = HEADLESS ? 200_000 : 1_000_000;
 export const TARGET_NETWORK = process.env.TARGET_NETWORK ? process.env.TARGET_NETWORK : 'localhost';
-export const WEB3TORRENT_URL = process.env.WEB3TORRENT_URL
+export const APP_URL = process.env.WEB3TORRENT_URL
   ? process.env.WEB3TORRENT_URL
   : 'http://localhost:3000';
```

Comment:
It might be slightly confusing that `APP_URL` uses `process.env.WEB3TORRENT_URL`, rather than `process.env.APP_URL`.

Reply:
That's fair, but this is how I rationalise it: Previously we had `APP_URL = http://localhost:3000` as an implicit default. Now that is an explicit default but with an override if `WEB3TORRENT_URL` is defined.

If we have cause to introduce, e.g.,  `RPS_URL`, (which maybe we should do soon) we could alter this line to do something sensible.

Open to other suggestions, though. Might be worth merging this PR and revisiting when we get to that particular bridge.